src/utils/index.es

```diff
@@ -0,0 +1,13 @@
+import React, { forwardRef } from 'react';
+
+export const getDisplayName = (wrapper, Component) => {
+  const name = Component.displayName || Component.name || 'Component';
+  return `${wrapper}(${name})`;
+};
+
+export const getForwardedComponent = (displayName, Component) => {
+  const WrappedComponent = (props, ref) => <Component {...props} forwardedRef={ref} />;
+  WrappedComponent.displayName = displayName;
```

Comment:
get `displayName` from `Component`. You assign it in each parent function anyway.

Reply:
Thats not the Component you think